WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderImageScanner.java

```diff
@@ -81,6 +77,32 @@ public ReaderImageList getImageList(int minImageWidth) {
     }
 
     /*
+     * returns true if there at least `minImageCount` images in the post content that are at
+     * least `minImageWidth` in size
+     */
+    public boolean hasUsableImageCount(int minImageCount, int minImageWidth) {
```

Comment:
I wrote that as a separate routine as an optimization, but I think you're right that the code duplication isn't ideal. I rewrote `hasUsableImageCount` to simply rely on `getImageList` in fcf24f0